web/src/main/java/com/epam/esm/controller/ApiVersion.java

```diff
@@ -0,0 +1,5 @@
+package com.epam.esm.controller;
+
+public enum ApiVersion {
+    V1
```

Comment:
why your api have only first version?
Every implemented task it's a new version